foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -484,6 +484,15 @@ def requeue_processor_job(last_job: ProcessorJob) -> None:
         elif new_ram_amount == 12288:
             new_ram_amount = 16384
 
+    # The AFFY pipeline is somewhat RAM-sensitive.
+    # Try it again with an increased RAM amount, if possible.
+    new_ram_amount = last_job.ram_amount
```

Comment:
This is going to overwrite the `new_ram_amount` that was set in the previous block for SALMON jobs.